src/vscode-bicep/src/commands/insertResource.ts

```diff
@@ -35,11 +30,25 @@ export class InsertResourceCommand implements Command {
 
     await this.client.sendNotification(insertResourceRequestType, {
       textDocument:
-        this.client.code2ProtocolConverter.asTextDocumentIdentifier(document),
+        this.client.code2ProtocolConverter.asTextDocumentIdentifier(editor.document),
       position: this.client.code2ProtocolConverter.asPosition(
         editor.selection.start
       ),
       resourceId: resourceId,
     });
   }
 }
+
+async function findOrShowEditor(documentUri?: Uri) {
+  if (!documentUri) {
+    const editor = vscode.window.activeTextEditor;
+    if (editor?.document.languageId !== 'bicep') {
```

Comment:
Should this check be performed even if a `documentUri` is present? I'm not sure if the change made to the `bicep.insertResource` command in the package.json file would prevent the command from being run on non-Bicep files or just hide it.